http/cookie.ts

```diff
@@ -49,6 +49,8 @@ export interface Cookie {
   secure?: boolean;
   /** The cookie's `HTTPOnly` attribute. If `true`, the cookie cannot be accessed via JavaScript. */
   httpOnly?: boolean;
+  /** The cookie's `Partitioned` attribute. If `true`, the cookie will be only be included in the `Cookie` request header if the domain it is embedded by matches the domain the cookie was originally set from.  */
```

Comment:
Can you add some warning message that this isn't standardized yet, and this option may change in the future (without following the semver semantics of the package)

ref: https://github.com/fastify/fastify-cookie/commit/9e4652ff303c9b1c4581c36a86d37b026a767bc4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R131

Reply:
Can you please add `@default {false}` to this?

Reply:
Addressed both in 802c56a, hopefully phrased properly.